packages/http/src/Adaptor.js

```diff
@@ -5,9 +5,8 @@ import { request as sendRequest, xmlParser } from './util';
  * Options provided to the HTTP request
  * @typedef {Object} RequestOptions
  * @public
- * @property {object|string} body - body data to append to the request. JSON will be converted to a string (but a content-type header will not be attached to the request).
  * @property {object} errors - Map of errorCodes -> error messages, ie, `{ 404: 'Resource not found;' }`. Pass `false` to suppress errors for this code.
- * @property {object} form - Pass a JSON object to be serialised into a multipart HTML form (as FormData) in the body.
+ * @property {boolean} form - True/False if the body data is multipart HTML form (as FormData).
```

Comment:
I would just say `True/False if the body data is multipart HTML form (as FormData).`